fideslib/db/base_class.py

```diff
@@ -143,9 +144,11 @@ def all(cls: Type[T], db: Session) -> list[T]:
         return db.query(cls).all()
 
     @classmethod
-    def filter(cls: Type[T], db: Session, *, conditions: list[Any]) -> Query:
+    def filter(
+        cls: Type[T], db: Session, *, conditions: BinaryExpression | BooleanClauseList
+    ) -> Query:
         """Fetch multiple models from a database table."""
-        return db.query(cls).filter(*conditions)
+        return db.query(cls).filter(conditions)
```

Comment:
I removed the unpacking of conditions here to better match the type. I have also confirmed that doing this fixes and error I was seeing when using the filter.

Reply:
that makes sense, I don't think the unpacking would actually have worked.